corporate/lib/activity.py

```diff
@@ -211,23 +211,11 @@ def get_remote_activity_plan_data(
     )
 
 
-def get_realms_with_default_discount_dict() -> Dict[str, Decimal]:
-    realms_with_default_discount: Dict[str, Any] = {}
-    customers = (
-        Customer.objects.exclude(default_discount=None)
-        .exclude(default_discount=0)
-        .exclude(realm=None)
-    )
-    for customer in customers:
-        assert customer.realm is not None
-        realms_with_default_discount[customer.realm.string_id] = assert_is_not_none(
-            customer.default_discount
-        )
-    return realms_with_default_discount
-
-
-def estimate_annual_recurring_revenue_by_realm() -> Dict[str, int]:  # nocoverage
+def get_estimated_arr_and_rate_by_realm() -> Tuple[Dict[str, int], Dict[str, str]]:  # nocoverage
+    # NOTE: Customers without a plan might still have a discount attached to them which
+    # are not included in `plan_rate`.
```

Comment:
This would be basically a stale `discount` percent display value? I guess we could do a migration to clear those if it was a priority.

Reply:
No, we don't want to do anything here. I just wanted to note that even if there is no discount displayed for the customer in the support table, that doesn't mean there is no discount applied to them. I am not sure where to best reflect that.

Reply:
Okay, I guess that won't be valid after https://github.com/zulip/zulip/pull/29957#discussion_r1602225763